### PR TITLE
Add Node.js version of rcloadenv.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: nodejs/src/index.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+/**
+ * Retrieves all variables in the given config.
+ *
+ * @param {string} configName
+ * @param {object} [opts]
+ * @returns {Promise}
+ */
+exports.getVariables = (configName, opts = {}) => {
+  opts.projectId || (opts.projectId = process.env.GCLOUD_PROJECT || process.env.GOOGLE_CLOUD_PROJECT);
```
Comment (tswast) [inline nodejs/src/index.js]: I think the Python and Go implementations have this flipped (prefer GOOGLE_CLOUD_PROJECT var over GCLOUD_PROJECT)

**Review by tswast — COMMENTED** [review]
Could you add testing instructions for the Node.js version to https://github.com/GoogleCloudPlatform/rcloadenv/blob/master/TESTING.md?

**Comment by jmdobry** [comment]
The Node.js implementation now supports the following:

```
$ rcloadenv --help

Wrap execution of the given command with runtime-config variables.
Usage: rcloadenv <configName> [options] -- [args...]

Options:
  --debug, -d                 Print debugging information.                                                     [boolean]
  --except, -E                If provided, a list of of runtime-config variables to exclude, otherwise all
                              runtime-config variables will be used.                                             [array]
  --only, -O                  If provided, a list of of runtime-config variables to load, otherwise all runtime-config
                              variables will be used.                                                            [array]
  --override, -o              Determines the behavior of rcloadenv in the case when a runtime-config variable conflicts
                              with a variable already available in the environment. If true, a runtime-config variable
                              will override the existing environment variable, otherwise the existing environment
                              variable will be kept.                                           [boolean] [default: true]
  --projectId, --project, -p  The project where the specified config is located.                                [string]
  --help                      Show help                                                                        [boolean]

Examples:
  rcloadenv my-config -- bash -c 'echo $MY_VARIABLE_NAME'       Print a variable's value from the specified config.
  rcloadenv my-config --only var1 var2 -- node app.js           Only load two variables from runtime-config.
  rcloadenv my-config --except var1 var2 -- node app.js         Load all but two variables from runtime-config.
  rcloadenv my-config --debug -- node app.js                    Print debugging information while loading variables.
  rcloadenv my-config --projectId my-project-id -- node app.js  Specify the project ID to use when loading variables.

For more information, see https://github.com/GoogleCloudPlatform/rcloadenv
```

**Review by tswast — APPROVED** [review]
Thank you!

**Comment by jmdobry** [comment]
Published: https://www.npmjs.com/package/@google-cloud/rcloadenv

File: nodejs/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "node": ">=4"
+  },
+  "bin": {
+    "rcloadenv": "./bin/rcloadenv"
+  },
+  "main": "src/index.js",
+  "scripts": {
+    "lint": "semistandard"
+  },
+  "dependencies": {
```
Comment (tswast) [inline nodejs/package.json:16]: I'm getting circle set up, and I noticed semistandard isn't listed anywhere. Should we add a devDependencies section?

Comment (jmdobry) [reply nodejs/package.json:16]: Either that or just make sure it's installed globally.

**Pull request merged**